packages/manager/cypress/integration/objectStorage/access-keys.spec.ts

```diff
@@ -1,16 +1,16 @@
-import { makeTestLabel } from '../../support/api/common';
-import { createBucket } from '../../support/api/objectStorage';
+import { createBucket } from 'support/api/objectStorage';
 import {
   containsVisible,
   fbtClick,
   fbtVisible,
   getClick,
   getVisible,
-} from '../../support/helpers';
+} from 'support/helpers';
+import { randomLabel } from 'support/util/random';
 
 describe('access keys', () => {
   it('create access key', () => {
-    const bucketLabel = 'cy-test-' + makeTestLabel();
+    const bucketLabel = 'cy-test-' + randomLabel();
```

Comment:
I may be wrong here, but doesn't `randomLabel()` always return a string starting with `cy-test-`? If so, could this be clean up?

```suggestion
    const bucketLabel = randomLabel();
```

Reply:
Nope, you're absolutely right! That const is definitely being assigned some wacky value like `cy-test-cy-test-fdksfk` -- I meant to circle back and fix that and totally forgot, thank you for catching it!